ci/build-linux.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+if [ "${TRAVIS_OS_NAME}" != "linux" ]; then
+  echo "Not a Linux-based build, exit successfully."
+  exit 0
+fi
+
+readonly IMAGE="cached-${DISTRO?}-${DISTRO_VERSION?}"
+readonly latest_id=$(sudo docker inspect -f '{{ .Id }}' ${IMAGE?}:latest 2>/dev/null || echo "")
```

Comment:
If it's a constant, the variable should be all-uppercase.

Reply:
Done.